src/hooks/useCounter.ts

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+export const useCounter = (
+	initialRating: number
+): {
+	counter: number;
+	increment: (c: number) => void;
+	decrement: (c: number) => void;
+} => {
+	const [counter, setCounter] = useState<number>(initialRating);
+
+	const increment = (c: number) => {
+		if (c === 5) {
+			return;
+		}
+		setCounter(c + 0.5);
+	};
+	const decrement = (c: number) => {
+		if (c === 0.5) {
```

Comment:
I don't remember, but I thought we agreed that the minimum rating value would be 0? correct me if I'm wrong though

Reply:
You're right I think! I'll change it to 0 and we can always change it back later if need be!